src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/resource/SyncResource.kt

```diff
@@ -116,10 +84,9 @@ class SyncResource(
       ),
     ],
   )
-  fun syncDelete(
-    @Schema(description = "Non-association ID", example = "1234123423", required = true)
-    @PathVariable
-    id: String,
+  fun sync(
```

Comment:
Isn't this complaining that the 2 functions are the same? (I'm looking this in GH so maybe I'm missing something)

Hang on, this is using method overloading or something isn't it? I see 👌

Reply:
yes overloading